test/forms/inputs/checkbox-group.test.js

```diff
@@ -96,7 +124,12 @@ test('CheckboxGroup passes down props to children', () => {
       className: 'custom-input-class',
     },
   }
-  const wrapper = mount(<CheckboxGroup {...props} />)
-  expect(wrapper.find('input.custom-group-class').exists()).toBe(false)
-  expect(wrapper.find('input.custom-input-class').exists()).toBe(true)
+  const { container } = render(<CheckboxGroup {...props} />)
+
+  // make sure fieldset has custom-class
+  const checkboxGroup = screen.getByRole('group', { name: 'Test Group' })
+  expect(checkboxGroup).toHaveClass('custom-group-class')
+
+  // make sure there is no class of custom-class within fieldset
+  expect(container.getElementsByClassName('custom-input-class').length).toBe(2)
```

Comment:
The comment and the assertion don't align. If there are no custom-input-class elements in the fieldset, wouldn't we expect length `toBe(0)`?

Reply:
Again, docs recommend not using `container`. Is there an alternative approach to explore here?

Reply:
Updated.